app/controllers/spaces_controller.rb

```diff
@@ -1,11 +1,11 @@
 class SpacesController < ApplicationController
 
   def index
-    @spaces = Space.all # all spaces, available or not
+    @spaces = Space.all # all spaces, available or not; TODO: filter 'available' + map
```

Comment:
you mean a map of all spaces that is filtered on the status = available?
Although I agree, we haven't managed this status yet; am I right?
I would suggest to filter the map on the location of the user BUT we don't have that either...

Reply:
Feel free to make that choice when you implement the map (or leave it out altogether); completely optional :)